app/views/commontator/comments/_show.html.erb

```diff
@@ -111,3 +108,13 @@
                                 id: comment.id } %>
   </div>
 </div>
+<%= render partial: 'commontator/comments/delete_comment_confirmation_modal' %>
+<script>
+  $(document).ready(function() {
+    $("#deletecommentModal").on("show.bs.modal", function(e) {
+        let href = $(e.relatedTarget).data('href');
+        $(e.currentTarget).find('#projects-comment-delete-button').attr("href",
+        href.toString());
+    });
+  });
```

Comment:
As we are moving from jQuery, can you rewrite this part in vanilla js ?

Reply:
Hello, It looks like vanilla js doesn't work with bootstrap 4.6 (currently being used in CircuitVerse afaik), One Solution is to update to bootstrap 5 but that would break other parts of code. another would be creating a new modal for it. I will look into this more tomorrow 
![Screenshot from 2023-11-13 22-06-33](https://github.com/CircuitVerse/CircuitVerse/assets/102225113/1252633e-c5d3-41c5-ad21-8afa1f628e61)

Reply:
Oh, I got it; I think rather than making it complex, we can use JQuery for now until we upgrade to latest bootstrap